doc/source/dev/cwl.md

```diff
@@ -0,0 +1,8 @@
+CWL import in Galaxy
+
```

Comment:
```suggestion
1. List paths to CWL tools in `tool_conf.xml`.
2. Enable the beta formats in `galaxy.yml`.
```

Reply:
@mvdbeek Is `enable_beta_workflow_format` actually needed? Can't find any related code.

Reply:
It might not be, I just copied this from the test script.